gulpfile.js

```diff
@@ -39,32 +39,48 @@ gulp.task('npminstall', () =>
     .pipe(es.mapSync(file => npmInstallTask(file.path)))
 );
 
-gulp.task('scanner:download', () =>
-  download(scanner.url)
-    .pipe(decompress())
-    .pipe(gulp.dest(paths.build.scanner))
+gulp.task('scanner:download', () => {
```

Comment:
My little understanding of gulp is that you should create 2 subtasks, and make this task depends on the two sub tasks. If you don't, since "download" is asynchronous, you may end up having only one completed.

Reply:
Or use es.merge as below

Reply:
Thanks, that was it! I'm surprised the packaging task didn't fail when files to copy were not present on the disk.